web/sites/default/config/views.view.group_nodes.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
     - field.storage.node.field_service_location_type
     - field.storage.node.field_start_date
     - node.type.calendar_item
+    - node.type.city_charter
```

Comment:
Still references code and charter as group content in the group content admin view.